src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
     $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}")##
 #end
 #macro(tp $n $p)##
-    $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p)##
+    #set($prf="GoogleApps.GoogleAppsConfigClass_")
+    #if($n.startsWith("googleapps"))#set($prf="")#end
+    #set($msg=$services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p))##
+    $stringtool.replacePattern($msg, '\[\[([^>]*)>>([^\]]*)\]\]', '<a href="$2">$1</a>')##
+#end
+#macro (displayInput $property)##
+    #set ($output = $doc.display($property, 'edit', $obj))##
+    #set ($output = $stringtool.removeStart($output, '{{html clean="false" wiki="false"}}'))##
+    #set ($output = $stringtool.removeEnd($output, '{{/html}}'))##
+    $output##
 #end
-$services.localization.render('googleapps.config.explanation')
 
+$services.localization.render('googleapps.config.explanation')
 
-{{html wiki=true}}
+{{html wiki=false}}
 
 <form id="$formId" method="post" action="$xwiki.getURL($configDoc, 'saveandcontinue')" class="xform">
-; $configDoc.display("activate", 'edit', $obj) <label for="${prefix}_activate">#t("activate")</label>
+<dl>
+    <dt>#displayInput("activate") <label for="${prefix}_activate">#t("activate")</label></dt>
+</dl>
```

Comment:
To have a valid HTML code block, the `<dl>` (description list) should contains both a `<dt>`(description term) and a `<dd>` (description details). There should a single `<dl>`  in your form, wrapping pairs of `<dt>` and `<dd>`.

Reply:
This made me add an empty `dd`. Fortunately, it hasn't bothered. This was the only place, apparently.